compare/README.md

```diff
@@ -5,11 +5,12 @@ To build a test utility configure libass with the `--enable-compare` flag.
 The utility works with `png` image files so there is external dependency of libpng.
 
 Test program command line:  
-`compare ([-i] <input-dir>)+ [-o <output-dir>] [-s <scale:1-8>] [-p <pass-level:0-3>]`
+`compare ([-i] <input-dir>)+ [-o <output-dir>] [-s <scale:1-8>[x<scale:1-8>]] [-p <pass-level:0-3>]`
 
 * `<input-dir>` is a test input directory, can be several of them;
 * `<output-dir>` if present sets directory to store the rendering results;
-* `<scale>` sets an oversampling factor (positive integer up to 8, default 1);
+* `<scale>` sets an oversampling factor or a pair of independent factors for horizontal and vertical oversampling
+  (in any case each factor is a positive integer up to 8, default 1x1);
```

Comment:
If you’re worried about `-s N` seeming like `-s Nx1`, perhaps it would be best to explictly state that a single value `NN` acts like `NxN`, or give an example like further down in the “Usage” text?

Reply:
Thanks for the feedback. I’ve tweaked the text further, and I think it’s significantly clearer now:

```
* `<scale>` sets factors for horizontal and vertical oversampling (positive integers up to 8)
  or a single factor to be used in both directions (default 1);
```